src/app/controllers/componentController.js

```diff
@@ -1,54 +1,105 @@
+${htmlLang !== undefined ? `{% set htmlLang = "${htmlLang}" %}` : ''}
+${pageTitleLang !== undefined ? `{% set pageTitleLang = "${pageTitleLang}" %}` : ''}
+${mainLang !== undefined ? `{% set mainLang = "${mainLang}" %}` : ''}
+${assetPath !== undefined ? `{% set assetPath = "${assetPath}" %}` : ''}
+${assetUrl !== undefined ? `{% set assetUrl = "${assetUrl}" %}` : ''}
+${themeColor !== undefined ? `{% set themeColor = "${themeColor}" %}` : ''}
+${bodyClasses !== undefined ? `{% set bodyClasses = "${bodyClasses}" %}` : ''}
+${containerClasses !== undefined ? `{% set containerClasses = "${containerClasses}" %}` : ''}
+${pageTitle !== undefined ? `{% block pageTitle %}${pageTitle}{% endblock %}` : ''}
+${headIcons !== undefined ? `{% block headIcons %}${headIcons}{% endblock %}` : ''}
+${head !== undefined ? `{% block head %}${head}{% endblock %}` : ''}
+${bodyStart !== undefined ? `{% block bodyStart %}${bodyStart}{% endblock %}` : ''}
+${skipLink !== undefined ? `{% block skipLink %}${skipLink}{% endblock %}` : ''}
+${header !== undefined ? `{% block header %}${header}{% endblock %}` : ''}
+${mainClasses !== undefined ? `{% set mainClasses = "${mainClasses}" %}` : ''}
+${main !== undefined ? `{% block main %}${main}{% endblock %}` : ''}
+${beforeContent !== undefined ? `{% block beforeContent %}${beforeContent}{% endblock %}` : ''}
+${content !== undefined ? `{% block content %}${content}{% endblock %}` : ''}
+${footer !== undefined ? `{% block footer %}${footer}{% endblock %}` : ''}
+${bodyEnd !== undefined ? `{% block bodyEnd %}${bodyEnd}{% endblock %}` : ''}
+`
+
+const govukComponentNunjucks = (component, params) => `{% from '${getComponentIdentifier(component)}/macro.njk' import ${component} %}{{${component}(${params})}}`
+
 module.exports = async (req, res, org) => {
-  const {
-    body = {},
-    params: {
-      version,
-      component
-    }
-  } = req
-
-  const { label, minimumSupported, paths } = orgs[org]
-
-  if (parseFloat(version) < minimumSupported) {
-    res.status(500).send(`This version of ${label} is not supported`)
-  } else {
-    await getNpmDependency(label, version)
-  
-    const params = JSON.stringify(body, null, 2)
-    try {
-      const nunjucksString = `{% from '${getComponentIdentifier(component)}/macro.njk' import ${component} %}{{${component}(${params})}}`
-      res.send(nunjucks(paths).renderString(nunjucksString))
-    } catch (err) {
-      res.status(500).send(err)
+    const {
+        body = {},
+        params: {
+            version,
+            component
+        }
+    } = req
+
+    const {label, minimumSupported, paths} = orgs[org]
+
+    if (parseFloat(version) < minimumSupported) {
+        res.status(500).send(`This version of ${label} is not supported`)
+    } else {
+        await getNpmDependency(label, version)
+
+        const params = JSON.stringify(body, null, 2)
+        try {
+            const nunjucksString = component === 'govukTemplate'
```

Comment:
I'm pretty sure this should be a different endpoint rather than a switch on the behaviour of this endpoint.